schemas/context/experienceevent-media.example.1.json

```diff
@@ -144,14 +161,21 @@
       "xdm:resumes": {
         "xdm:value": 0
       },
+      "xdm:estimatedStreams": {
+        "xdm:value": 2
+      },
+      "xdm:pauseImpactedStreams": {
+        "xdm:value": 3
+      },
       "xdm:pauses": {
         "xdm:value": 0
       },
       "xdm:pauseTime": {
         "xdm:value": 0
       },
       "xdm:idp": "MyTVOnline",
-      "xdm:mediaAuth": "Sample auth"
+      "xdm:mediaAuth": "Sample auth",
+      "xdm:dayPart": "daytime"
```

Comment:
What does this property represent?

Reply:
It specifies the time of the day when the content was broadcast or played.